### PR TITLE
[14.0][MIG] website_event_snippet_calendar : Migration to v14

### PR DESCRIPTION
**Pull request by chandni299**
1. Migration to v14
2. I made a small improvement in displaying not only start but end date as well. See the screenshot below how it looks.

calendar_event_list_with_date_end

Do you guys think it is ok to commit this with the migration PR or is it better to create a separate [IMP] PR one for this minor change?

### CODE REVIEW

File: website_event_snippet_calendar/static/src/xml/snippets.xml [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
             <div class="row">
                 <div class="col-12 event_details">
                     <span class="date">
                         <t t-esc="event.date_begin" />
                     </span>
+                    <span>
+                        <i class="fas fa fa-arrow-right text-muted mx-1 my-1" />
+                    </span>
+                    <span class="date">
+                        <t t-esc="event.date_end" />
```
Comment (chienandalu) [inline website_event_snippet_calendar/static/src/xml/snippets.xml]: Could we make it optional? Maybe some users don't want to show them both

Comment (chandni299) [reply website_event_snippet_calendar/static/src/xml/snippets.xml]: ok. Optional you mean exactly where and how you want to show it?

Comment (chienandalu) [reply website_event_snippet_calendar/static/src/xml/snippets.xml]: Well, more whether or not you should show it

Comment (chandni299) [reply website_event_snippet_calendar/static/src/xml/snippets.xml]: @chienandalu, you can review the changes, to make it optional for users to show/hide the end date of events.

File: website_event_snippet_calendar/controllers/main.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def days_with_events(self, start, end):
         type="json",
         website=True,
     )
-    def events_for_day(self, day=None, limit=None):
+    def events_for_day(self, day=None, limit=None, enddateFlag=None):
```
Comment (chienandalu) [inline website_event_snippet_calendar/controllers/main.py]: OCA's convention is to name variables with snake case (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#342variable-names)

**Review by chienandalu — COMMENTED** [review]
Little picky comment:

**Comment by chienandalu** [comment]
@Tardo please can you add your review

File: website_event_snippet_calendar/controllers/main.py (hunk 2 above)
Comment (chandni299) [reply website_event_snippet_calendar/controllers/main.py]: Correct it. Thanks!

Comment (chandni299) [reply website_event_snippet_calendar/controllers/main.py]: Improved it. Thanks!

**Comment by chandni299** [comment]
Could we open this pull request so we will have reviews and later merge this PR @chienandalu @Tardo ?

**Comment by pedrobaeza** [comment]
I reopen it, but we are not interested anymore in this addon.

**Comment by chandni299** [comment]
> I reopen it, but we are not interested anymore in this addon.

Thanks for the quick response. Understood.
@pedrobaeza . Could you please share the reason why we're no longer interested in this addon? It would help me

**Comment by pedrobaeza** [comment]
We are not using it in any of our customers.

**Pull request closed without merging**